src/api/models/precomputed.js

```diff
@@ -29,7 +29,12 @@ export default async db => {
     };
 
     collection.delete = async id => {
-        await db.collection(`pc_${id}`).drop();
+        try {
+            await db.collection(`pc_${id}`).drop();
+        } catch {
+            // Collection does not exist, no big deal
+            console.warn(`Failed to drop collection 'pc_${id}'`);
+        }
```

Comment:
This circumvents `MongoServerError: ns not found` errors I got, which seem to be consecutive to https://github.com/Inist-CNRS/lodex/pull/1860. @touv feel free to suggest a better implementation if you disagree with this one! :slightly_smiling_face: